mavenproject/src/main/java/mtm/Deck.java

```diff
@@ -11,6 +11,6 @@ public void addCard(Card card) {
 	
 	public void printCards() {
 		for(Card card : cards) 
-			card.Message();
+			System.out.println(card.toString());
```

Comment:
I don't think classes other than main should print to `stdout`. Maybe add logging instead. This is fine for now, but change it later.